CGPs/cgp-0143.md

```diff
@@ -0,0 +1,186 @@
+- **Close Relationships with RWA and Stablecoin Builders**: Artemis works closely with key leaders in the Real World Assets (RWA) and Stablecoin space, such as Castle Island Ventures, BlueChip.org, and Standard & Poor’s. These relationships enable Artemis to stay at the forefront of developments in the industry, ensuring that our data are in the hands of the key opinion leaders of the space.
+- **Most Cited Institutional Data Provider in the Space**:
+Artemis Citations:
+- Trusted by key Celo stakeholders already: https://x.com/RegenRene/status/1807826195884696040 & https://x.com/MentoLabs/status/1794988382197571920
+
+# Funding Request
+Artemis offers comprehensive solutions with the Application Activity Monitor and Stablecoin Dashboard, priced based on the equivalent cost of hiring 2-3 full-time employees (FTEs) per year to build and maintain similar data infrastructure as well as what other ecosystems are paying for Artemis’ services.
+
+- **Application Activity Monitor**: $150,000 per year
+- **Stablecoin Dashboard**: $124,000 per year
+
+Our pricing model reflects the extensive expertise and ongoing support required to ensure these tools are continuously updated, reliable, and user-friendly.
+
+**Total Funding Request**: $274,000 per year
+
+**Multisig Signers**: Add wallet address here about who will be on the multisig that receives the funding
+
+# Conclusion
+Integrating Artemis into the Celo ecosystem will significantly enhance transparency, usability, and adoption of Celo’s blockchain technology, especially towards liquid token investors and large financial institutions–the core customers of Artemis and potential investors of the Celo ecosystem. Many of these customers are not aware of all the new developments of Celo as they understand Celo as a climate focused chain. Artemis can help provide investors an up-to-date, unbiased and data-backed view on the latest developments of Celo including its transition to an Ethereum L2 so that Celo remains top-of-mind in the investor community.
+
+By providing comprehensive data solutions such as the Application Activity Monitor and Stablecoin Dashboard, Artemis will also empower the Celo community with valuable insights and drive informed decision-making. This partnership aligns with Celo’s mission of financial inclusion and stability, supporting the ecosystem’s growth and long-term success.
+
+## Proposed Changes
+ 
+Fill out the following template for each transaction in the proposal
+ 
+1. Description of transaction 1
+  - Destination: A human readable description of the address and method being called
+  - Data: A human readable description of the transaction data
+  - Value: How much CELO is being sent, and why?
+ 
+## Verification
```

Comment:
You can take an example from here and leave proposal ID as XXX https://github.com/celo-org/governance/blob/main/CGPs/cgp-0140.md